src/zjs_test_promise.c

```diff
@@ -7,15 +7,33 @@
 #include "zjs_promise.h"
 #include "zjs_util.h"
 
+typedef struct dummy_handle {
+    uint32_t value;
+} dummy_handle_t;
+
+static void post_promise(void* handle)
+{
+    if (handle) {
+        dummy_handle_t* h = (dummy_handle_t*)handle;
+        if (h->value != 42) {
```

Comment:
Thanks for the test. But move this constant definition up to a #define or something to make it more clear.